src/index.css

```diff
@@ -478,3 +489,46 @@ ACCESSIBILITY - Also see any other focus styles
   outline: 2px solid rgba(0, 121, 116, 1);
   outline-offset: 1px;
 }
+
+/*
+=================
+PRINTING STYLES
+=================
+*/
+
+/* `margin: 0` at bottom working without print,
+    not working for print. Why? */
+@media print {
+  .header_segment,
+  #feedback_fixed,
+  #reset_fixed,
+  #form-nav,
+  #section-bottom,
+  #predictionsForm,
+  #resultsIntro,
+  #App .tabular.menu,
+  .print-row {
+    display: none;
+  }
+
+  #cliff-effects-tool,
+  #App .very.padded.segment {
+    padding: 0;
+  }
+
+  #App .bottom.segment {
+    border: 0;
+  }
+
+  #root #App .ui.segment {
+    margin: 0;
+  }
+
+  @page {
+    size: portrait;
+  }
+
+  #App {
+    filter: grayscale(100%);
```

Comment:
I feel like we don't necessarily have to make it black-and-white; people can print in black and white if they want to anyway. And some printers might sneakily still use color ink even if printing B&W content in color mode. But not a big deal.

Reply:
If the setting is automatically set to print in black & white, then that's fine and we can get rid of a style. I'll do a bit of research on it to make sure.